### ISSUE TITLE
False positive on css:S4666 (selectors should not be duplicated)

### ISSUE THREAD

**Issue by alasdairhurst** [SonarSource/sonar-css#294]
Not sure if this is intentional or not but there's the case where you intentionally have duplicate selectors when separating styles that apply to multiple classes from styles that apply to specific classes:

I got this false positive for my scss file that was linted with sonarqube 7.4 (who knows if the rules are up to date, i really hope so)

```
.myclass, .myotherclass {
  font-size: 10px;
  < lots more styles >
}
// particular override for myclass
.myclass {
  color: red;
}
// particular override for myotherclass
.myotherclass {
  color: blue;
}
```

Sonar is complaining that .myclass is a duplicate selector, first used on the line with .myclass, .myotherclass
